packages/web/src/components/nav/desktop/PlaylistLibrary/CollectionNavItem.module.css

```diff
@@ -9,6 +9,7 @@
   display: flex;
   align-items: center;
   line-height: normal;
+  overflow: hidden;
```

Comment:
why was this needed? just curious

Reply:
oh i see your comment now :) yes as discussed not needed, thanks!